### PR TITLE
trash-cli: add page

### PR DESCRIPTION
**Pull request by cfsanderson**
- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by agnivade** [comment]
These seem to have different sub-commands. Ideally, we should have separate pages for each of the commands. And just link them from the main page.

**Comment by cfsanderson** [comment]
🤔 not sure what you mean @agnivade. `trash-cli` is the name of the software but isn't a command per se.
https://github.com/andreafrancia/trash-cli

**Comment by agnivade** [comment]
I thought this was something like git. `git` is the main command, but we have separate pages for `git-push`, `git-commit` and so on. Do each of the commands like `trash-put` and `trash-empty` have flags and other details which need to be mentioned ?

File: pages/common/trash-cli.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# trash-cli
+
+> Trash-cli is a less permanent way of removing files via the command line.
```
Comment (agnivade) [inline pages/common/trash-cli.md]: "less permanent" seems a bit unclear. Perhaps - "A command line interface to the trashcan APIs" sounds better.

File: pages/common/trash-cli.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Trash files and directories:
+
+`trash-put {{file_name}}`
+
+- Empty the trashcan(s):
```
Comment (agnivade) [inline pages/common/trash-cli.md]: So does this mean there can be multiple trashcans ? How do we control which trash can to dump a file to ?

**Comment by sbrl** [comment]
Maybe we should have a page for each command then? I'm not sure.

**Comment by agnivade** [comment]
Ideally we should, but then each page will have just one example. So maybe the ease of use outweighs the need for consistency ?

Also, another question is will the user search for something like `tldr trash-put`, `tldr trash-rm` ? Then certainly we need separate pages because if they are all under `trash-cli`, then it won't return any result. But if they just search for `trash-cli`, then I think it should be fine to have it in a single page.

ping @waldyrious - thoughts ?

File: pages/common/trash-cli.md (hunk 1 above)
Comment (cfsanderson) [reply pages/common/trash-cli.md]: Sounds good.

File: pages/common/trash-cli.md (hunk 2 above)
Comment (cfsanderson) [reply pages/common/trash-cli.md]: No, I'll get rid of that

**Pull request merged**